### PR TITLE
Moves the Medieval Sec Armor kit to the Sec Wardrobe, and fixes a bug related to the wardrobe.

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Kawaii-Big-Boss
tweak: Relocated the Medieval Sec Armor to the SecDrobe
fix: Fixed a bug where malformed security uniforms were displayed on the SecDrobe, sometimes for sale too.
fix: That means I removed Jumpskirts. They're dead now.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
benis

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Tested and works, no problem. No amaris here!

File: hippiestation/code/modules/vending/security.dm [hunk 1]
```diff
@@ -1,2 +1,3 @@
-/obj/machinery/vending/security
-	hippie_products = list(/obj/item/storage/box/seclarp = 4)
\ No newline at end of file
+/obj/machinery/vending/wardrobe/sec_wardrobe
+	icon = 'hippiestation/icons/obj/vending.dmi'
+	hippie_products = list(/obj/item/storage/box/seclarp = 4, /obj/item/clothing/under/rank/security/skirt = -100)
```
Comment (praisenarsie) [inline hippiestation/code/modules/vending/security.dm:3]: ...

**Comment by Pyko1** [comment]
should be a cargo pack really, or some maint phatlewts.

Or maybe even a forge item or something.

**Comment by Carbonhell** [comment]
Is this a meme

**Comment by ghost** [comment]
lol
securiskirts can actually go though tbh
+1

**Comment by JohnGinnane** [comment]
are there not already in?

**Comment by Manly-Dwarf** [comment]
No, they're just in the SecTech. Back when they were added we didn't have a SecDrobe.

**Comment by bardofrage** [comment]
Removes jumpskirts, + 1 billion

**Comment by Carbonhell** [comment]
I get making a meme changelog but it is a bit too vague, specify what your change does please
Tweak: jumpskirts removed
Or something like that

**Comment by Carbonhell** [comment]
Also +1

**Review by Carbonhell — APPROVED** [review]

**Comment by Manly-Dwarf** [comment]
okay

**Comment by Carbonhell** [comment]
@Kawaii-Big-Boss did you test it?not sure if going into negative values will make it behave correctly

**Comment by Manly-Dwarf** [comment]
`Tested and works, no problem. No amaris here!`

**Comment by JohnGinnane** [comment]
@Carbonhell negative values for hippie vending is how we lower/remove items from the products list

**Comment by Carbonhell** [comment]
Ah alright

**Pull request merged**